setup.py

```diff
@@ -51,10 +51,10 @@ def main():
     install_requires = ['py>=1.4.29']  # pluggy is vendored in _pytest.vendored_packages
     extras_require = {}
     if has_environment_marker_support():
-        extras_require[':python_version=="2.6" or python_version=="3.0" or python_version=="3.1"'] = ['argparse']
+        extras_require[':python_version=="2.6"'] = ['argparse']
         extras_require[':sys_platform=="win32"'] = ['colorama']
     else:
-        if sys.version_info < (2, 7) or (3,) <= sys.version_info < (3, 2):
+        if sys.version_info < (2, 7):
```

Comment:
This might be as well `sys.version_info[:2] == (2, 6)`, for consistency with the marker usage in `54`